.circleci/config.yml

```diff
@@ -632,6 +648,7 @@ jobs:
             [ -d data ] || mkdir data && chmod 777 data
       - docker_login
       - make_docker_images
+      - install_and_setup_gcloud
```

Comment:
gcloud is already installed, what problem have you noticed?

Reply:
It fails with openssl error. I saw in your childchain pr you resolved it by updating docker-compose, I resolved it this way

Reply:
okay, so lets resolve it "the same way" :)
so that we don't wonder what it is or what the reason for it is.
consistency.